<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoverWidget</class>
 <widget class="QWidget" name="CoverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{color: rgb(137,129,161);font: 12px &quot;微软雅黑&quot;;}

</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>290</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
		color:white;
        border: none;
        text-align: center;
        background: rgb(128, 51, 128);
}
QProgressBar::chunk {
		color:white;
        background: rgb(83, 61, 138);
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="format">
    <string>计算中...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="hideBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(83, 61, 138);
color: rgb(255, 255, 255);
border-radius:12px;
font:12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>隐藏</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>90</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>还未同步至最新区块，当前余额可能不准确。

此时发出的交易将不被网络接受。</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>250</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>上一区块年龄</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>220</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>已同步区块高度</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>220</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取中...</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>250</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取中...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
